# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../qmail-autoresponder/without-mysql.diff
# Copyright (C) 2006 The T2 SDE Project
# Copyright (C) 1998 - 2004 ROCK Linux Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- ./Makefile.orig	2004-05-06 19:52:42.000000000 -0400
+++ ./Makefile	2004-05-06 19:52:54.000000000 -0400
@@ -61,7 +61,7 @@
 main.o: compile main.c qmail-autoresponder.h
 	./compile main.c
 
-programs: qmail-autoresponder qmail-autoresponder-mysql
+programs: qmail-autoresponder
 
 qmail-autoresponder: load qmail-autoresponder.o main.o
 	./load qmail-autoresponder main.o -lbg-iobuf -lbg-str
--- ./insthier.c.orig	2004-05-06 19:52:58.000000000 -0400
+++ ./insthier.c	2004-05-06 19:53:03.000000000 -0400
@@ -8,7 +8,6 @@
   int man1;
 
   c(bin,  "qmail-autoresponder",       -1, -1, 0755);
-  c(bin,  "qmail-autoresponder-mysql", -1, -1, 0755);
 
   man1 = d(man, "man1", -1, -1, 0755);
   c(man1, "qmail-autoresponder.1",     -1, -1, 0644);
